src/styles/blocks/card.scss

```diff
@@ -0,0 +1,134 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  min-width: 1024px;
+  margin: auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under the element selector.